lnwallet/chanfunding/psbt_assembler.go

```diff
@@ -628,5 +630,6 @@ func isSegWitScript(pkScript []byte) bool {
 	}
 
 	return parsed.Class() == txscript.WitnessV0PubKeyHashTy ||
-		parsed.Class() == txscript.WitnessV0ScriptHashTy
+		parsed.Class() == txscript.WitnessV0ScriptHashTy ||
```

Comment:
I think we can just use this function instead: https://pkg.go.dev/github.com/btcsuite/btcd/txscript#IsWitnessProgram

Reply:
Ah yes, makes sense, fixed.